launcher/minecraft/skins/SkinDelete.h

```diff
@@ -18,17 +18,8 @@
 
 #pragma once
 
-#include "net/NetRequest.h"
+#include <net/NetRequest.h>
```

Comment:
Not a fan of this: usually if an include is surrounded by < > it should be a library, not a file from the project. Can you revert back to using " ?